keyboards/gray_studio/space65/keymaps/via/space65.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
As with the Think 6.5, I don't know if it's such a good idea to include the VIA json in the QMK repo. If it needs to be updated for any reason, you now have to make two PRs to two different repos, otherwise they'll get out of sync.

Reply:
Resolved by 2d69475